plugins/hls-cabal-plugin/test/Main.hs

```diff
@@ -220,10 +223,108 @@ codeActionTests = testGroup "Code Actions"
+                                   , ("AWin32Console", "1.19.1")
+                                   ]
+    , testHiddenPackageSuggestions "Check CabalAdd's parser, no version, unicode comma"
+                                   [ "It is a member of the hidden package \8216base\8217"
+                                   , "It is a member of the hidden package \8216Blammo-wai\8217"
+                                   , "It is a member of the hidden package \8216BlastHTTP\8217"
+                                   , "It is a member of the hidden package \8216CC-delcont-ref-tf\8217"
+                                   , "It is a member of the hidden package \8216AAI\8217"
+                                   , "It is a member of the hidden package \8216AWin32Console\8217"
+                                   ]
+                                   [ ("base", T.empty)
+                                   , ("Blammo-wai", T.empty)
+                                   , ("BlastHTTP", T.empty)
+                                   , ("CC-delcont-ref-tf", T.empty)
+                                   , ("AAI", T.empty)
+                                   , ("AWin32Console", T.empty)
+                                   ]
+    , testHiddenPackageSuggestions "Check CabalAdd's parser, with version, unicode comma"
+                                   [ "It is a member of the hidden package \8216base-0.1.0.0\8217"
+                                   , "It is a member of the hidden package \8216Blammo-wai-0.11.0\8217"
+                                   , "It is a member of the hidden package \8216BlastHTTP-2.6.4.3\8217"
+                                   , "It is a member of the hidden package \8216CC-delcont-ref-tf-0.0.0.2\8217"
+                                   , "It is a member of the hidden package \8216AAI-0.1\8217"
+                                   , "It is a member of the hidden package \8216AWin32Console-1.19.1\8217"
+                                   ]
+                                   [ ("base","0.1.0.0")
+                                   , ("Blammo-wai", "0.11.0")
+                                   , ("BlastHTTP", "2.6.4.3")
+                                   , ("CC-delcont-ref-tf", "0.0.0.2")
+                                   , ("AAI", "0.1")
+                                   , ("AWin32Console", "1.19.1")
+                                   ]
+    , expectFailBecause "TODO fix regex for these cases" $
+      testHiddenPackageSuggestions "Check CabalAdd's parser, with version, unicode comma"
+                                   [ "It is a member of the hidden package \82163d-graphics-examples\8217"
+                                   , "It is a member of the hidden package \82163d-graphics-examples-1.1.6\8217"
+                                   ]
+                                   [ ("3d-graphics-examples", T.empty)
+                                   , ("3d-graphics-examples", "1.1.6")
+                                   ]
```

Comment:
I think these should be moved to a different file, since they are simple unit tests.